docs/Makefile

```diff
@@ -0,0 +1,23 @@
+MAN_PAGE=man/man1/dmd.1
+DMD=dmd
+DMD_ROOT=..
+G=$(DMD_ROOT)/generated
+
+################################################################################
+# Main targets
+################################################################################
+
+build: $(MAN_PAGE)
+
+preview:
+	man -l $(MAN_PAGE)
+
+################################################################################
+# Actual compilation commands
+################################################################################
+
+$(MAN_PAGE): $G/gen_man
+	$< > $@
```

Comment:
The main mainfile can call this target to regenerate the man page when `cli.d` changed, but as I said before, I think the discussion whether that's a good idea or not, shouldn't block this PR and thus belongs into another PR.